changelog.md

```diff
@@ -1,6 +1,24 @@
+## Changes from 1.4.10 to 1.4.11
+
+### Fixed issues
+
+- [MARATHON-7992](https://jira.mesosphere.com/browse/MARATHON-7992) - Validation errors on root group in large clusters can lead Marathon to crash.
```

Comment:
I'm looking at https://github.com/mesosphere/marathon/compare/v1.4.10...releases/1.4

I see MARATHON-7990 but not MARATHON-7992

Reply:
It is for 1.4.11

Reply:
looks like slack is down :( .   The issue is in the tracking (and not on your end).   Marathon-7992 is Unassigned and in review however it has #5874 in the comments.   #5874 is merged (so not in review) and lists MARATHON-7990 as it's JIRA issue.

I'll approve but under duress :)

Reply:
Yeah, my bad. At least I added a PR link in the comments, though? :)

Reply:
It is a tiny thing, not even worth mentioning. Merging :)